algorithms/ar-lreg/javascript/linear_regression.js

```diff
@@ -0,0 +1,49 @@
+// Using the Least Squares Method
+function linearRegression (points) {
+  const n = points.length;
+  const x_points = points.map(([x, _]) => x);
```

Comment:
Codacy found an issue: [Identifier 'x_points' is not in camel case.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6390369)